pkgs/development/interpreters/guile/2.0.nix

```diff
@@ -22,7 +22,7 @@
 
   depsBuildBuild = [ buildPackages.stdenv.cc ]
```

Comment:
Should pkgsBuildBuild be used here?

Reply:
It would have to be `pkgsBuildBuild.targetPackages.stdenv.cc`, so ehh hardly worth it for now.